device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/https/HttpsConnection.java

```diff
@@ -182,12 +182,21 @@ public void setRequestHeader(String field, String value)
      *
      * @param timeout the read timeout.
      */
-    public void setReadTimeoutMillis(int timeout)
+    public void setReadTimeout(int timeout)
```

Comment:
Doesn't this change a public interface?

Reply:
Java doesn't consider this class part of the public interface, no.